app/policies/dish_policy.rb

```diff
@@ -0,0 +1,18 @@
+class DishPolicy < ApplicationPolicy
+  def index?
+    true
+  end
+
+  def create?
+    user.present?
+  end
+
+  def update?
+    return true if user.present? && user == record.user
```

Comment:
Wait. It is dish policy. record is object by Dish class. You don't have user relation in dish model, do you?